### PR TITLE
Fix outbound link to Cilium K8s Installation Guide

### PR DESCRIPTION
**Pull request by elvinjgalarza**

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ policies using an example application.
 ## Deploying Cilium for Production Use
 
 For detailed instructions around deploying Cilium for production, see:
-[Cilium Kubernetes Installation Guide](https://docs.cilium.io/en/stable/kubernetes/intro/)
+[Cilium Kubernetes Installation Guide](https://docs.cilium.io/en/stable/concepts/kubernetes/intro/)
```
Comment (spzala) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:75]: @elvinjgalarza thanks for fixing this link and welcome to the community! 
/lgtm

Comment (elvinjgalarza) [reply content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:75]: of course! thank you for the opportunity :)

**Review by Arhell — COMMENTED** [review]
@elvinjgalarza
pls sign the CLA in order for the PR to be reviewed

**Comment by elvinjgalarza** [comment]
hi @Arhell, thank you for the update. It looks like the check for CLA was successful. I also checked The Linux Foundation and the site is saying I'm "authorized to contribute code to this project through your membership with CNCF - IBM". 🤔

**Comment by spzala** [comment]
> hi @Arhell, thank you for the update. It looks like the check for CLA was successful. I also checked The Linux Foundation and the site is saying I'm "authorized to contribute code to this project through your membership with CNCF - IBM". 🤔

Interesting.. it shows both cla yes and no label. Allowing retest to see if that updates anything.

**Comment by spzala** [comment]
Thanks @fejta and @bradtopol

**Review by bradtopol — COMMENTED** [review]
verified that new link works properly.

**Comment by bradtopol** [comment]
New link works and renders properly. Great job!
/lgtm
/approve

**Pull request merged**